### PR TITLE
Mac install issues banner (bug 1124243)

### PR DESCRIPTION
**Pull request by mstriemer**
Depends on https://github.com/mozilla/marketplace-core-modules/pull/17.

screenshot 2015-01-22 15 36 49

### CODE REVIEW

**Comment by mstriemer** [comment]
Going to make a change to how dismissing works.

**Comment by ngokevin** [comment]
I feel like it could use a smidge more top/bottom padding.

File: src/templates/_includes/os_x_banner.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<mkt-banner id="mac-install-notice" success>
```
Comment (ngokevin) [inline src/templates/_includes/os_x_banner.html]: i think i prefer classes due to the potential CSS specificity issues using IDs may introduce (even though we're not styling it here)

**Comment by chuckharmston** [comment]
11 words of text on the button? Surely we can do better than that, it barely looks like a button at all.

**Comment by ngokevin** [comment]
Probably good to have an e2e test, esp. for the dismissing behavior. Might have to do some mocking magic for the capabilities though...

**Comment by mstriemer** [comment]
The dismissing behaviour is from `<mkt-banner>`. If we're going to test it I think marketplace-elements is the right spot for that.

**Comment by mstriemer** [comment]
I'm not changing the dismiss behaviour, I didn't like it. I'll revisit it next week or when it gets sufficiently annoying, whichever is shorter most likely.

**Comment by mstriemer** [comment]
Merged but comments are still welcome.

**Pull request merged**